### PR TITLE
fix CI setup

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

**Comment by bastelfreak** [comment]
I would like to get this reviewed and merged. Afterwards we can increase the strictness of prospector and remove some of the disabled checks.

File: marmoset/webserver/pxe.py [hunk 1]
```diff
@@ -46,10 +47,7 @@ def post(self):
         """
         args = parser.parse_args()
 
-        if args.persistent in ['true', 'True']:
-            persistent = True
-        else:
-            persistent = False
+        persistent = True if args.persistent in ['true', 'True'] else False
```
Comment (mmerfort) [inline marmoset/webserver/pxe.py]:
```python
persistent = args.persistent in ['true', 'True']
```

File: marmoset/installstatus/db_helper.py [hunk 2]
```diff
@@ -23,19 +24,21 @@ def _connect(cls):
...
+        """
+        Tests if the 'installstatus' table exists in DB, else creates it.
 
         :param con: sqlite3 connection
         """
         result = con.execute("SELECT name FROM sqlite_master "
                              "WHERE type = 'table' AND name = 'installstatus'")
         result = result.fetchall()
-        if len(result) == 0:
+        if result:
```
Comment (mmerfort) [inline marmoset/installstatus/db_helper.py]: This should probably be
```python
if not result:
```

**Review by mmerfort — APPROVED** [review]

File: marmoset/installstatus/__init__.py [hunk 3]
```diff
@@ -51,8 +51,4 @@ def get_stats(args):
 
 def is_uuid(uuid):
     """Check if provided string is a UUID"""
-    if validation.is_uuid(uuid):
-        return True
-    else:
-        print('Invalid uuid %s.' % uuid)
-        return False
+    return True if validation.is_uuid(uuid) else False
```
Comment (JanKoppe) [inline marmoset/installstatus/__init__.py]: This seems a bit redundant. Shouldn't this work just as well?

```
return validation.is_uuid(uuid)
```

**Review by JanKoppe — APPROVED** [review]
Tiny cosmetic thing, but overall looks like a good cleanup.

File: marmoset/installstatus/__init__.py (hunk 3 above)
Comment (bastelfreak) [reply marmoset/installstatus/__init__.py]: thanks, fixed it

**Pull request merged**